source/_integrations/sensehat.markdown

```diff
@@ -140,8 +140,35 @@ group:
   kitchen:
     - group.sense_hat
 ```
-
-### Directions for installing on Raspberry Pi Raspbian Based installation:
+## Installation
+### On Home Assistant Operating System
+
+Things got complicated for poor _SenseHAT_ with Home Assistant Operating System. I found finally a way to use it.
+
+- Follow instructions [here](https://www.home-assistant.io/hassio/enable_i2c/) to enable i2c on Home Assistant Operating System
+- Install "Custom deps deployment" add-on. To do so
+    - add ``https://github.com/home-assistant/hassio-addons-development`` to your add-ons repositories
+    - select *Custom deps deployment* and enable it
+- Configure "Custom deps deployment" add-on
+    ```yaml
+    pypi:
+    - >-
+      git+https://github.com/RPi-Distro/RTIMULib#egg=rtimulib&subdirectory=Linux/python
+    apk:
+    - gcc
+    - g++
+    - py3-sensehat
+    - i2c-tools-dev
+    - python3-dev
+    - raspberrypi-dev
+    - raspberrypi-libs
+    ```
+- Start the plugin and check the logs to make sure everything is fine. Set it to "Start at boot"
+- Reboot and enjoy your lights
+
+Steps verified with Home Assistant Operating System 4.15. At the moment of this writing, sensors of the SenseHat are still not working, but at least now they can be seen.
```

Comment:
We don't document versions. Documentation should match the latest codebase.

Reply:
Solved

Reply:
However I find confusing and having a negative impact the fact of not documenting version, if you do not write the date of the documentation. In this specific case, the documentation is 3 years old (and the previous text mention a version) and we have at least 3 git issues on this item without solution (until now) and few others on the forum on the same subject. Maybe you can show in the right column not only the version since a module is maintained, but the last date of modification of the page. That would really help.

Reply:
We don't do that. Our documentation reflects the latest version and that is it.
At this point in time, keeping up with the current state of affairs is already hard enough. Maybe if we master that.

However, out of scope for this PR.

Reply:
Thanks